plugins/jenkins-backend/config.d.ts

```diff
@@ -38,11 +38,16 @@ export interface Config {
        * instance.
        */
       name: string;
-
       baseUrl: string;
       username: string;
       /** @visibility secret */
       apiKey: string;
+      extraRequestHeaders?:
+        | {
+            /** @visibility secret */
+            Authorization?: string;
```

Comment:
Your previous commit had both lowercase and uppercase variants declared - i still think that was probably a good idea! It's a bit frustrating that we can't safely cover _any_ case mix, but it's at least very unlikely that someone gives `auTHORization` and suddenly is leaking secrets.